### ISSUE TITLE
Text is mirrored/flipped

### ISSUE THREAD

**Issue by phourniner** [CorellanStoma/CreArts-Discord#11]
Hello, when using your theme I noticed the text is flipped or mirrored. Example here:  
![image](https://user-images.githubusercontent.com/73620435/125023213-fc953680-e043-11eb-8cc0-f2b83320fc07.png) 
I am also using the plugin "Staff Tags" which is also being affected as shown in the screenshot, "MOD" is backwards.

I am wondering if this is intentional behavior, as I understand the text may be mirroring to get that "messaging effect" with the speech bubble on the right side when you are talking in a server. I do enjoy this feature, as it's easier to see when I post in a server, and aesthetically I enjoy it.

Thank you and I hope this can be fixed!

**Comment by CorellanStoma** [comment]
Yeah, I can definitely fix that, in fact everything is mirrored and then individual elements are remirrored again to get that messaging effect.

This can lead to a few bugs if not all classes have been mirrored individually.

The Staff Tags plugin I do not know, I assume that it is a Powercord plugin, plugins I have to adapt for the theme but is not a big effort.

**Comment by OneTrackMinded** [comment]
Here is another version of the same issue, except with the username 
![image](https://user-images.githubusercontent.com/67457515/126439760-c6ba4f6c-dc05-4361-b49f-0b6258417a35.png)

**Comment by mcsuchtiyt** [comment]
This happens with Threads too
![bugreport](https://user-images.githubusercontent.com/41999358/129491829-057c2a70-9e8f-4dcd-9cfa-59bc10e56800.png)

**Comment by CorellanStoma** [comment]
Outdated Bug from CreArts v2.
